core/controllers/acl_decorators.py

```diff
@@ -2753,6 +2753,84 @@ def test_can_accept_suggestion(
     return generate_decorator_for_handler
 
 
+def can_update_suggestions(handler):
+    """Decorator to check whether user can update the suggestions that
+    they are allowed to review.
+
+    Args:
+        handler: function. The function to be decorated.
+
+    Returns:
+        function. The newly decorated function that has common checks and
+        permissions specified by passed in decorator. Users with rights
+        to accept translations andthe user who created a particular
+        translation are allowed to perform this action.
```

Comment:
Typo for 'and the'.

Also, the second sentence seems like an incomplete thought/fragment--can you update it to be a bit clearer?

Reply:
Done